content/en/user-manual/publishing/mobile/gonative.md

```diff
@@ -0,0 +1,15 @@
+---
+title: GoNative
+layout: usermanual-page.hbs
+position: 2
+---
+
+[GoNative][gonative] is a paid third party service that can take a URL and create a native mobile application that uses a native WebView to run in the mobile in a few clicks.
```

Comment:
"to run in the mobile"?

Reply:
Ah, this was meant to be 'run on mobile'. Changing now